<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBook</class>
 <widget class="QWidget" name="AddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>78</width>
    <height>48</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="widgetTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fictionBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Non-fiction</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fiction</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="genreBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Biography</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Academic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Historical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Journalism</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Travel Guide</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Guides</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="widgetTitle_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Page Count</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pgEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dewey Decimal</string>
   </property>
  </widget>
  <widget class="QSlider" name="ddSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ddCounter">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dayEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>01</string>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Release Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>01</string>
   </property>
  </widget>
  <widget class="QLabel" name="widgetTitle_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
